kubernetes/annotations_and_labels.md

```diff
@@ -8,6 +8,9 @@ This page defines common annotations and labels we set in Kubernetes objects.
   location. For now this should be in our `giantswarm/giantswarm` repository in
   which we crosslink all pages to create a reasonable documentation of all kinds
   of Kubernetes resources, their functionality and relationships.
+- `giantswarm.io/restart` - this annotation is meant to be used for DaemonSet,
```

Comment:
Here we should explain what actions this annotation imposes. Where it is used should be described in one of the sections at the bottom of the page.

Reply:
I feel that the structure lower down in this document doesn't really work for this annotation. This is kind of generic and can be put on any object that is wanted to be updated. Do you have ideas how to change that structure or should this be just described here?

Reply:
Now when you explained what this is supposed to do in the other comment I think it's fine. Let's improve the comment a bit to try to reduce the general confusion.